openassessment/xblock/static/js/spec/lms/oa_staff_area.js

```diff
@@ -310,23 +315,23 @@ describe('OpenAssessment.StaffAreaView', function() {
             verifyFocused(closeButton);
         };
 
-        it('shows the correct buttons with no panels initially', function() {
+        it('shows the correct buttons when full grading is not enabled', function() {
             var view = createStaffArea(),
                 $buttons = $('.ui-staff__button', view.element);
             expect($buttons.length).toBe(2);
+            expect($buttons).toHaveAttr('aria-expanded', 'false');
             expect($($buttons[0]).text().trim()).toEqual('Manage Individual Learners');
             expect($($buttons[1]).text().trim()).toEqual('View Assignment Statistics');
-            expect(getVisibleStaffPanels(view).length).toBe(0);
```

Comment:
Note to reviewers: the initial visibility is governed by the CSS, and for a Jasmine test, the CSS is not in play. Therefore we can't test this anymore.